projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cds/merchandising-carousel.e2e-spec.ts

```diff
@@ -1,28 +1,32 @@
 import * as merchandisingCarousel from '../../../helpers/vendor/cds/merchandising-carousel';
```

Comment:
General note:

- e2e sometimes takes 7 mins to complete

- give consents part of the e2e fails always
- without consent part of the e2e fails at times (not always)

------
I was wondering maybe we can split the e2e test, but this will be the last resort for later

Reply:
FYI for `give consents part of the e2e fails always` I have noticed this as well. Looking into it now to see if it is CDS or spartacus or CCV2 who changed something